src/App.jsx

```diff
@@ -18,6 +19,7 @@ function App() {
           <Route path="contact" element={<Contact />} />
           <Route path="login" element={<LoginPage />} />
           <Route path="signup" element={<Signup />} />
+          <Route path="userportal" element={<UserPortal />} />
```

Comment:
There's already a route for this (you'll see it once you rebase your branch and grab the changes from development) so feel free to remove this.

Reply:
With the existing thing already there, you just need to insert your new component onto that page 😁